neo/Prompt/Commands/Invoke.py

```diff
@@ -331,7 +345,6 @@ def test_invoke(script, wallet, outputs, withdrawal_tx=None,
             # print("Used %s Gas " % engine.GasConsumed().ToString())
 
             consumed = engine.GasConsumed() - Fixed8.FromDecimal(10)
-            consumed = consumed.Ceil()
```

Comment:
https://docs.neo.org/docs/en-us/tooldev/advanced/tx_execution.html#b02a262b
> Assign Math.max(0, GasConsumed - 10).Ceiling() to InvocationTransaction.Gas as transaction's system fee. Note that system provides 10 Gas for free in each InvocationTransaction.

Reply:
@hal0x2328 could you record your findings for GAS costs from Slack here for discussion? It would appear that either, the current implementation of neo-python (with `Ceil` rounding) is correct per the above documentation, or the documentation is outdated and no longer being utilized in neo-cli.

Reply:
The documentation seems to match the code: https://github.com/neo-project/neo/blob/56174b8fc2e31bc0f300e4dab1f61bd8cf81ca6e/neo/Network/P2P/Payloads/InvocationTransaction.cs#L44-L46

Reply:
Okay. In that case, I will remove this change.